pages/docs/sharing-data-externally/email-reports.mdx

```diff
@@ -0,0 +1,58 @@
+# E-mail Reports
```

Comment:
Nit: `email` seems more common than `e-mail` nowadays? It's also the language we choose in our app.